### ISSUE TITLE
Hyp login getting failed

### ISSUE THREAD

**Issue by mohammad-quanit** [hypermodeinc/hyp-cli#35]
I have installed hyp cli, and when I try to log in using the `hyp login` command, it redirects me to the auth callback page and shows a success message, but it fails on the terminal. Due to that, I am unable to use any AI model provided by hyper mode. Below is the exact error message I am seeing on the terminal.

```
Opening login page...
✔ Please select an organization: mquanit
Successfully logged in as m***************m! 🎉
node:fs:448
    return binding.readFileUtf8(path, stringToFlags(options.flag));
                   ^

Error: ENOENT: no such file or directory, open '/Users/mquanit/.hypermode/settings.json'
    at Module.readFileSync (node:fs:448:20)
    at readSettingsJson (file:///Users/mquanit/.hypermode/cli/dist/util/index.js:103:24)
    at LoginIndex.writeToEnvFile (file:///Users/mquanit/.hypermode/cli/dist/commands/login/index.js:146:26)
    at Server.<anonymous> (file:///Users/mquanit/.hypermode/cli/dist/commands/login/index.js:95:34) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/mquanit/.hypermode/settings.json'
}

Node.js v23.0.0
```

OS: Mac OS venture
Hypermode: @hypermode/hyp-cli/0.1.2
nodejs: v23.0.0
go: go1.23.0 darwin/arm64

**Comment by jairad26** [comment]
Hi @mohammad-quanit , sorry about the bug this is now fixed & released in v0.1.3

if you reinstall the CLI it should be working now

**Comment by mohammad-quanit** [comment]
Thanks, it is fixed now.
